docs/api-conventions.md

```diff
@@ -232,67 +232,8 @@ TODO: Plugins, extensions, nested kinds, headers
 HTTP Status codes
 -----------------
 
-The following HTTP status codes may be returned by the API.
-
-#### Success codes
-
-* `200 StatusOK`
-  * Indicates that the request completed successfully.
-* `201 StatusCreated`
-  * Indicates that the request to create kind completed successfully.
-* `204 StatusNoContent`
-  * Indicates that the request completed successfully, and the response contains no body.
-  * Returned in response to HTTP OPTIONS requests.
-
-#### Error codes
-* `307 StatusTemporaryRedirect`
```

Comment:
Please don't delete all this. We just added it. :-(  We get questions about what the precise behavior is. This gives us a place to elaborate, provide examples, etc. Also, some of it is really specific to Kubernetes, such as usage of resourceVersion.